pkg/replication/replication_mode.go

```diff
@@ -232,46 +232,44 @@ func (m *ModeManager) loadDRAutoSync() error {
 	return nil
 }
 
-func (m *ModeManager) drSwitchToAsyncWait(availableStores []uint64) error {
+func (m *ModeManager) drSwitchToAsyncWait(availableStores []uint64) {
```

Comment:
Why change it?

Reply:
The caller always ignored this error, and all errors were logged within the function, so I removed it.

Reply:
But the tests will check the return value of `drSwitchToAsync`.

Reply:
Got it. I will revert it for the test